mod_reforged/hooks/experimental_modules/onSpawnEntity.nut

```diff
@@ -47,6 +62,11 @@ local spawnEntity = ::Tactical.spawnEntity;
 	// The single parameter is necessary as this function is used in Time.scheduleEvent
 	q.RF_onSpawn <- { function RF_onSpawn( _ )
 	{
+		if (this.m.__RF_HasOnSpawnBeenCalled)
+			return;
+
+		this.m.__RF_HasOnSpawnBeenCalled = true;
+
```

Comment:
What does this relate to?
Is this a different fix or a new feature?
The commit only talks about fixing something for the player specifically

Reply:
The code that does the player fix will also lead to an edge case where if an actor that was already spawned was removed from map and then replaced on map will trigger `RF_onSpawn` again. So, we need this check as part of that fix to control that edge case.